pkg/apis/externaldns/types.go

```diff
@@ -28,47 +28,51 @@ var (
 
 // Config is a project-wide configuration
 type Config struct {
-	Master         string
-	KubeConfig     string
-	Sources        []string
-	Namespace      string
-	FqdnTemplate   string
-	Compatibility  string
-	Provider       string
-	GoogleProject  string
-	DomainFilter   string
-	Policy         string
-	Registry       string
-	TXTOwnerID     string
-	TXTPrefix      string
-	Interval       time.Duration
-	Once           bool
-	DryRun         bool
-	LogFormat      string
-	MetricsAddress string
-	Debug          bool
+	Master             string
+	KubeConfig         string
+	Sources            []string
+	Namespace          string
+	FqdnTemplate       string
+	Compatibility      string
+	Provider           string
+	GoogleProject      string
+	DomainFilter       string
+	AzureConfigFile    string
+	AzureResourceGroup string
+	Policy             string
+	Registry           string
+	TXTOwnerID         string
+	TXTPrefix          string
+	Interval           time.Duration
+	Once               bool
+	DryRun             bool
+	LogFormat          string
+	MetricsAddress     string
+	Debug              bool
 }
 
 var defaultConfig = &Config{
-	Master:         "",
-	KubeConfig:     "",
-	Sources:        nil,
-	Namespace:      "",
-	FqdnTemplate:   "",
-	Compatibility:  "",
-	Provider:       "",
-	GoogleProject:  "",
-	DomainFilter:   "",
-	Policy:         "sync",
-	Registry:       "txt",
-	TXTOwnerID:     "default",
-	TXTPrefix:      "",
-	Interval:       time.Minute,
-	Once:           false,
-	DryRun:         false,
-	LogFormat:      "text",
-	MetricsAddress: ":7979",
-	Debug:          false,
+	Master:             "",
+	KubeConfig:         "",
+	Sources:            nil,
+	Namespace:          "",
+	FqdnTemplate:       "",
+	Compatibility:      "",
+	Provider:           "",
+	GoogleProject:      "",
+	DomainFilter:       "",
+	AzureConfigFile:    "",
```

Comment:
That makes sense to me.  I'll make that change and update the tutorial to mention that it checks that location by default.